lib/Screens/specific_story.dart

```diff
@@ -0,0 +1,201 @@
+import 'dart:convert';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_html/flutter_html.dart';
+import 'package:http/http.dart' as http;
+import 'package:url_launcher/url_launcher.dart';
+
+import 'show_user.dart';
+import '../classes/comment.dart';
+import '../classes/news.dart';
+import '../functions/all_functions.dart';
+
+class SpecificNews extends StatefulWidget {
+  const SpecificNews({super.key, required this.news});
+
+  final Future<News> news;
+
+  @override
+  State<SpecificNews> createState() => _SpecificNewsState();
+}
+
+class _SpecificNewsState extends State<SpecificNews> {
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return RefreshIndicator(
+      onRefresh: _refresh,
+      color: Colors.amber,
+      child: Scaffold(
+        body: FutureBuilder(
+          future: widget.news,
+          builder: (context, snapshot) {
+            if (snapshot.hasData) {
+              final List<int> commentIDs = List<int>.from(snapshot.data!.kids);
+
+              return NotificationListener<ScrollEndNotification>(
+                onNotification: (notification) {
```

Comment:
This could probably be wrapped in a method for readability

```suggestion
                onNotification: _handleNotification,
```